src/client/services/feature/feature.js

```diff
@@ -61,7 +62,7 @@ var FeatureService = SparkBase.extend(
     return this.request({
       method: 'POST',
       api: 'feature',
-      resource: 'features/users/' + this.spark.device.userId + '/developer',
```

Comment:
I'm not sure.
I think it is something that the values are being set temporarily and then not stored correctly. This is why I only detected the error after resetting the app. How would I simulate and test this?